components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -815,7 +820,7 @@ export class Panel extends React.Component<Props, State> {
         walletType={externalWallet ? externalWallet.type : undefined}
         walletState={walletStatus}
         walletProvider={utils.getWalletProviderName(externalWallet)}
-        onVerifyClick={onVerifyClick}
+        onVerifyClick={this.onVerifyClick}
```

Comment:
Does it work to leave the existing behavior as-is? The previous behavior would take the user straight to the login page if they are in a "disconnected-verified" state, meaning that their access token expired. I think we still want that to happen?

Reply:
resolved